### PR TITLE
ENH more comprehensive README

### LINKED ISSUE #204: README needs more resources before 0.1.0-alpha
Our README file needs some love before alpha release
- [x] link to PyPI rather than test simple (in anticipation of release)
- [x] temporarily remove reference to paper for citing
- [x] write contributing guidelines and put a link to them in README
- [x] link to source code, PyPI, and issue tracker (like scikit-learn)

A good model is scikit-learn: https://github.com/scikit-learn/scikit-learn/blob/master/README.rst

### LINKED ISSUE #224: ENH README.md: install from source + install from pip
Hi,

I think we should add two manners to install the software in the README(similarly to pytorch)

1. Via the sources.(this is currently not proposed)
2. Via "pip"

Let me know your thoughts about it!

Edouard

### PR DESCRIPTION
**Pull request by edouardoyallon**
- [x] Closes #224 
- [x] Closes #204

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +13,24 @@ Use Kymatio if you need a library that:
...
 
-Website: [http://www.kymat.io](http://www.kymat.io)
-
 
 ## Installation
 
+### Important links
+
+* Official source code repo: [https://github.com/kymatio/kymatio](https://github.com/kymatio/kymatio)
+* Download releases: https://pypi.org/project/????????
```
Comment (lostanlen) [inline README.md]: there is a chicken-and-egg situation here. we to be registered on PyPI to put this on the README, and we need a README to register on PyPI.
I would be in favor of removing this line and adding it on the beta

File: README.md [hunk 2]
```diff
@@ -13,11 +13,24 @@ Use Kymatio if you need a library that:
...
+* Issue tracker: [https://github.com/kymatio/kymatio/issues](https://github.com/kymatio/kymatio/issues)
+* Website: [http://www.kymat.io](http://www.kymat.io)
+
+### Source code
+
+You can check the latest sources with the command:
+
+```
+git clone https://github.com/kymatio/kymatio.git
```
Comment (lostanlen) [inline README.md]: as stated earlier today, I would be in favor of removing this section

File: README.md [hunk 3]
```diff
@@ -54,7 +67,7 @@ The software uses PyTorch + NumPy FFT on CPU, and PyTorch + CuPy + CuFFT on GPU.
 
 If you use this code in your work please cite our paper:
 
-The scattering authors, [*Kymatio: Fast Scattering in 1-D,2-D,3-D*]()
+~~ The scattering authors, [*Kymatio: Fast Scattering in 1-D,2-D,3-D*]() ~~
```
Comment (lostanlen) [inline README.md]: The paper is not out yet, so I would be in favor of removing this line altogether

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  ```
+
+- If your issue is related to GPU acceleration, please copy-paste the output
+  from the [environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
+  of PyTorch.
+  You can get the script and run it with:
+  ```python
+  wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
+  # For security purposes, please check the contents of collect_env.py before running it.
+  python collect_env.py
```
Comment (lostanlen) [inline CONTRIBUTING.md]: Thanks Ed, this is awesome

**Comment by edouardoyallon** [comment]
Please do a PR! Let's end up this asap! 👍

File: README.md (hunk 3 above)
Comment (edouardoyallon) [reply README.md]: Alright! Feel free to PR

File: README.md (hunk 2 above)
Comment (edouardoyallon) [reply README.md]: Alright!

File: README.md (hunk 1 above)
Comment (edouardoyallon) [reply README.md]: We will add it in timethen!

File: CONTRIBUTING.md (hunk 4 above)
Comment (edouardoyallon) [reply CONTRIBUTING.md]: I just cced yours!:)

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+          $ git clone git@github.com:YourLogin/kymatio.git
+          $ cd kymatio
+
+3. Remove any previously installed versions of Kymatio:
+
+          $ pip uninstall kymatio
+
+and install your local copy with testing dependencies:
+
+          $ pip install -e .[tests]
```
Comment (lostanlen) [inline CONTRIBUTING.md]: this is incorrect and needs a fix

Comment (edouardoyallon) [reply CONTRIBUTING.md]: No worry, what is the goodway?

Comment (lostanlen) [reply CONTRIBUTING.md]: i'm making a 2nd pr to your branch re: contributing.md

Comment (lostanlen) [reply CONTRIBUTING.md]: see https://github.com/edouardoyallon/kymatio/pull/2

**Pull request merged**